webknossos/webknossos/dataset/mag_view.py

```diff
@@ -32,11 +33,15 @@
 from .view import View
 
 
-def _find_mag_path_on_disk(
-    dataset_path: Path, layer_name: str, mag_name: str, path: Optional[str] = None
+def _find_mag_path(
+    dataset_path: Path,
+    layer_name: str,
+    mag_name: str,
+    path: Optional[str | Path] = None,
```

Comment:
```suggestion
    path: Optional[Union[str, Path]] = None,
```

The Union notation you used is only supported since python 3.10, as we support python 3.9 as well we need the `Union[...]` notation.

Reply:
Oh nice, thanks for finding this :pray:

Reply:
Nice the ci is green now :tada: